Dockerfile

```diff
@@ -1,16 +1,14 @@
-FROM ruby:2.3.0
-RUN apt-get update -qq && apt-get install -y nodejs netcat
+FROM ruby:2.4.1
+RUN apt-get update -qq && apt-get install -y nodejs
 
-RUN mkdir /myapp
+ENV APP_ROOT /champaign-ak-processor
 
-WORKDIR /tmp
-COPY Gemfile Gemfile
-COPY Gemfile.lock Gemfile.lock
-RUN bundle install --jobs 4
+RUN mkdir $APP_ROOT
+ADD . $APP_ROOT
+WORKDIR $APP_ROOT
 
-EXPOSE 3000
-ADD . /myapp
-WORKDIR /myapp
+RUN bundle install --jobs 4 --deployment && bundle package
```

Comment:
+1 to `--deployment`, can't believe we didn't have that in our production images! :-) thanks

Reply:
Do we use this Dockerfile for production only? If so, we can also make it only install production gems?

Reply:
I've actually been using it for development still, but don't mind changing my development setup so it's the same as for Champaign (Redis and Postgres through Docker Compose and the rails app the vanilla way). That's why I left the test + development dependencies in - I also thought that it's not such a big application that it'd be a major deal but agree that it'd make sense to move them out eventually.